README.md

```diff
@@ -44,5 +44,5 @@ and can be used to install any pre-requisites.
    after the templates have been configured. You can use the environment variables `$SLAVES` to
    get a list of slave hostnames and `/root/spark-ec2/copy-dir` to sync a directory across machines.
       
-   e. Modify https://github.com/mesos/spark/blob/master/ec2/spark_ec2.py to add your module to
```

Comment:
I think you're looking for: https://github.com/apache/spark/blob/master/ec2/spark_ec2.py